tasks/node/agent/version.go

```diff
@@ -29,70 +25,51 @@ func (t NodeAgentVersion) Explain() string {
 // Dependencies - Returns the dependencies for this task.
 func (t NodeAgentVersion) Dependencies() []string {
 	return []string{
-		"Node/Config/Agent",
-		"Base/Log/Copy",
+		"Node/Env/Dependencies",
 	}
 }
 
 // Execute - The core work within this task
 func (t NodeAgentVersion) Execute(options tasks.Options, upstream map[string]tasks.Result) tasks.Result {
-	var result tasks.Result
-
-	if upstream["Node/Config/Agent"].Status != tasks.Success {
-		result.Status = tasks.None
-		result.Summary = "Node agent not detected"
-		return result
+	if upstream["Node/Env/Dependencies"].Status != tasks.Info {
```

Comment:
I removed our check for Node/Config/Agent because Node/Env/Dependencies already checks that Node/Config/Agent ran before it.